src/main.cc

```diff
@@ -64,6 +66,7 @@ int main(int argc, char* argv[]) {
   }
 
   ImgIoUtilWriteFile(output, webp_data.bytes, webp_data.size);
+  WebPPictureFree(current_frame);
```

Comment:
Each `WebPPicture` in `pics` will be freed, no need to free this `current_frame` (which is just an alias of the last element).